examples/simple_pio_example.c

```diff
@@ -33,33 +32,28 @@ if [ ! -e $HOME/build/geopm/lib/libgeopmpolicy.so ]; then
     make install
     cd ..
 fi
-gcc -I$HOME/build/geopm/include -L$HOME/build/geopm/lib -lgeopmpolicy simple_pio_example.c
-LD_LIBRARY_PATH=$HOME/build/geopm/lib:$LD_LIBRARY_PATH ./a.out
-# Total energy for package 0: 518.16 (joules)
-# END SCRIPT
+gcc -I$HOME/build/geopm/include -L$HOME/build/geopm/lib -lgeopmpolicy
+simple_pio_example.c LD_LIBRARY_PATH=$HOME/build/geopm/lib:$LD_LIBRARY_PATH
+./a.out # Total energy for package 0: 518.16 (joules) # END SCRIPT
```

Comment:
I think those comments were on a line by themself for a reason.  It's hard to see now.

Reply:
fixed with special comment